src/utils.ts

```diff
@@ -157,8 +157,8 @@ export function parseIndexOptions(indexSpec: IndexSpecification): IndexOptions {
     // {location:'2d', type:1}
     keys = Object.keys(indexSpec);
     keys.forEach(key => {
-      indexes.push(key + '_' + indexSpec[key]);
-      fieldHash[key] = indexSpec[key];
+      indexes.push(key + '_' + (indexSpec as any)[key]);
+      fieldHash[key] = (indexSpec as any)[key];
```

Comment:
Hi! Me again, hope I'm not bothering. Can you (in the interim, per the above conversation) fix needing to add the `as any` bit by iterating over the result of `Object.entries` instead? The following seems to make the transpiler happy:
```typescript
    Object.entries(indexSpec).forEach(([key, value]) => {
      indexes.push(key + '_' + value);
      fieldHash[key] = value;
    });

Reply:
Good suggestion! thanks :)